src/TaskRouter.cpp

```diff
@@ -31,28 +35,69 @@ bool RouterTask::setup(System &system) {
 bool RouterTask::loop(System &system) {
   // do routing
   if (!_fromModem.empty()) {
-    std::shared_ptr<APRSMessage> msg  = _fromModem.getElement();
-    String                       path = msg->getPath();
+    std::shared_ptr<APRSMessage> modemMsg  = _fromModem.getElement();
+    String path;
```

Comment:
do not define this object here if you just need it further down.

Reply:
soved?